pkg/instrumentors/bpf/github.com/gorilla/mux/probe.go

```diff
@@ -44,15 +44,15 @@ import (
 type Event struct {
 	StartTime   uint64
 	EndTime     uint64
-	Method      [100]byte
+	Method      [6]byte
```

Comment:
What about CONNECT and OPTIONS?

Reply:
Yeah I just realized this recently too. I had grabbed these changes from the [http/server probe](https://github.com/open-telemetry/opentelemetry-go-instrumentation/blob/main/pkg/instrumentors/bpf/net/http/server/probe.go#L48), but agree it should be higher. I can up to 7 or 10 or leave alone at 100?
Also I'd like to quick fix the http/server one here, should I do here or in a new PR?

Reply:
> I can up to 7 or 10 or leave alone at 100?

I think 7 is sufficient, right?

> Also I'd like to quick fix the http/server one here, should I do here or in a new PR?

either sounds good to me

Reply:
Actually, hm. This changes the http target to remove the leading forward slash. Let me look at this a bit more.